lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -223,23 +348,17 @@ def active(self) -> bool:
         # We don't do `"active" in state` because the word "active" is also part of "inactive" :)
         return "inactive" not in services[f"slurm.{self._service.value}"]
 
+    def type(self) -> ServiceType:
+        """Return the service type of the managed service."""
+        return self._service
 
-class ConfigurationManager:
-    """Control configuration of a Slurm component."""
+
+class _SnapConfigurationManager(ConfigurationManager):
```

Comment:
`_SnapConfigManager` :pleading_face::point_right::point_left:

 ```suggestion
class _SnapConfigManager(ConfigurationManager):
```